docs/references/javascript/organization-domain.mdx

```diff
@@ -0,0 +1,92 @@
+| `totalPendingSuggestions` | `number` | The number of total pending suggestions sent to emails that match the domain name. |
+| `createdAt` | `Date` | The date and time the membership was created. |
+| `updatedAt` | `Date` | The date and time the membership was updated. |
+
+### `OrganizationDomainVerification`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `status` | `'unverified' \| 'verified'` | The status of the verification process. |
+| `strategy` | `'email_code'` | A string that indicates strategy of the verification. |
+| `attempts` | `number` | A number that indicates how many attempts have occured in order to verify the domain. |
+| `expiresAt` | `Date` | The expiration date and time of the verification. |
+
+
+## Methods
+
+### `delete()`
+
+Deletes the organization domain and removes it from the organization.
+
+```typescript
+function delete: () => Promise<void>;
+```
+
+
+## `prepareAffiliationVerification()`
+
+```typescript
+function prepareAffiliationVerification(params: PrepareAffiliationVerificationParams): Promise<OrganizationDomain>;
+```
+
+Begins the verification process of a created organization domain. This is a required step in order to complete the registration of the domain under the organization.
+
+### `PrepareAffiliationVerificationParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `affiliationEmailAddress` | `string` | An email address that is affiliated with the domain name (e.g. user@example.com). |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<[OrganizationDomain][org-domain-ref]\></code> | This method returns a <code>Promise</code> which resolves with a <code>[OrganizationDomain][org-domain-ref]</code> object. |
+
+
+## `attemptAffiliationVerification()`
+
+```typescript
+function attemptAffiliationVerification(params: AttemptAffiliationVerificationParams): Promise<OrganizationDomain>;
+```
+
+Attempts to complete the domain verification process. This is a required step in order to complete the registration of a domain under an organization, as the administrator should be verified as a person who is affiliated with that domain.
+
+Make sure that an `OrganizationDomain` object already exists before you call this method, by first calling `OrganizationDomain.prepareAffiliationVerification`.
+
+### `AttemptAffiliationVerificationParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `code` | `string` | The one-time code that was sent to the user as part of this verification step. |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<[OrganizationDomain][org-domain-ref]\></code> | This method returns a <code>Promise</code> which resolves with a <code>[OrganizationDomain][org-domain-ref]</code> object. |
+
+
+[org-domain-ref]: /docs/references/javascript/organization-domain
+[enrollment-mode-ref]: /docs/organizations/verified-domains#enrollment-modes

```

Comment:
If we are going to link to this reference, the `manual_invitation` option needs to be added to the [reference page](https://clerk.com/docs/organizations/verified-domains). Because it says its an option here in the table, but the [current reference page](https://clerk.com/docs/organizations/verified-domains) does not list it as an option.

Reply:
I'd like this to be acknowledged!